x/docs/client/cli/tx.go

```diff
@@ -79,6 +93,23 @@ func getCmdShareDocument(cdc *codec.Codec) *cobra.Command {
 				Recipients: ctypes.Addresses{recipient},
 			}
 
+			if viper.GetBool(FlagSign) {
+				document.DoSign = &types.DocumentDoSign{
+					StorageURI:         viper.GetString(FlagSignStorageURI),
+					SignerInstance:     viper.GetString(FlagSignSignerInstance),
+					VcrID:              viper.GetString(FlagSignVcrID),
+					CertificateProfile: viper.GetString(FlagSignCertificateProfile),
+					SdnData: types.SdnData{
```

Comment:
All fields in snd_data are references to the x509 standard fields only. The values of the fields will be provided by another service after inserting the transaction in the chain. It's better provided only one flag `--sdn_data` with the references of fields comma separated. Example     
```
--sdn_data="common_name,surname,serial_number,given_name,organization,country"
```

Permitted fields references are 

- common_name
- surname
- serial_number
- given_name
- organization
- country

The value of flag `sdn_data` should be decomposed with the comma and inserted in an array structure